debian/Dockerfile

```diff
@@ -23,6 +22,7 @@ RUN buildDeps=' \
 		autoconf \
 		automake \
 		bzip2 \
+		curl \
```

Comment:
ah yeah and we run as read only anyways :)

On Wed, Apr 13, 2016 at 4:55 PM, Tianon Gravi <notifications@github.com>
wrote:

> In debian/Dockerfile
> <https://github.com/jfrazelle/irssi/pull/8#discussion_r59646062>:
>
> > @@ -23,6 +22,7 @@ RUN buildDeps=' \
> >  		autoconf \
> >  		automake \
> >  		bzip2 \
> > +		curl \
>
> Meh -- Alpine has it built into busybox, and it's generally useful, so I'm
> +1 on keeping it (especially since we're dropping curl)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jfrazelle/irssi/pull/8/files/669c2b52e5db58804eb373901b79c8e49826eb27#r59646062>
>



-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>